pkg/mcp/snapshot/snapshot.go

```diff
@@ -153,7 +155,8 @@ func (c *Cache) Watch(request *source.Request, pushResponse source.PushResponseF
 	cancel := func() {
 		c.mu.Lock()
 		defer c.mu.Unlock()
-		if info, ok := c.status[group]; ok {
+		var ok bool
+		if info, ok = c.status[group]; ok {
```

Comment:
It seems a little surprising to see a variable from another calling context re-used here. Did this fix a linter/compiler warning?

Reply:
yes. It was a golang warning. Reverting. Good catch.